src/OrchardCore.Themes/TheAdmin/Assets/js/TheAdmin/TheAdmin.ts

```diff
@@ -1,4 +1,3 @@
-import removeDiacritics from "./removeDiacritics";
 import "./menu";
 import "./resizeDetector";
 import * as bootstrap from "bootstrap";
```

Comment:
We need to create a separate typescript file for everything related with Bootstrap component that we use in OC. This way we don't import directly Bootstrap on TheAdmin.ts file and we can abstract the ConfirmModal component. We need to start doing more abstractions so that we can eventually swap Bootstrap with other components.

We also need to stop doing inline javascript in .cshtml files.